src/components/app/map/types.ts

```diff
@@ -13,10 +15,18 @@ export interface LocationMarkerProps {
   icon: string;
   activeMarkerId: number | null;
   setActiveMarkerId: (id: number | null) => void;
+  setMarkerRef: (marker: Marker | null, id: number) => void;
 }
 
 export interface GoogleMapProps {
   apiKey: string;
   locations: LocationMarker[];
   icon: string;
 }
+
+export type Location = {
```

Comment:
there is a `LocationMarker` interface present on the top of this file. Can't we re-use that instead of making a new one?